CHANGELOG.md

```diff
@@ -91,6 +91,16 @@
 - Bumped inspect from 3.1.2 to 3.1.3
   [#8589](https://github.com/Kong/kong/pull/8589)
 
+
+### Breaking Changes
+
+##### Configuration
+
+- Change the default of `lua_ssl_trusted_certificate` to `system`
+  [#8602](https://github.com/Kong/kong/pull/8602). If you are upgrading from 2.x and want this variable to keep
+  working as before, please manually set it to empty
+  (`lua_ssl_trusted_certificate = [nothing in here]`) before upgrading. 
```

Comment:
```suggestion
  [#8602](https://github.com/Kong/kong/pull/8602).
```

Is probably enough.

Reply:
Where do we usually mention these things?